specifyweb/frontend/js_src/lib/components/SchemaConfig/Components.tsx

```diff
@@ -33,7 +34,7 @@ export function SchemaConfigHeader({
         {schemaText.changeBaseTable()}
       </Button.Small>
       <span className="-ml-2 flex-1" />
-      <Button.Small onClick={handleSave}>{commonText.save()}</Button.Small>
+      <Submit.Save onClick={handleSave}>{commonText.save()}</Submit.Save>
```

Comment:
while this is a good change you did, keep in mind that submit buttons should not be used outside of `<Form>`. Please add `<Form>` element around the entire schema config page

Reply:
Do you mean around Container.Full line 109 in SchemaConfig/index.tsx?